devenv/docker/blocks/influxdb/docker-compose.yaml

```diff
@@ -1,5 +1,5 @@
   influxdb:
-    image: influxdb:1.7.6
+    image: influxdb:1.8.1-alpine
```

Comment:
Am i right that this is the only thing related to the issue title?  The rest is making things private

Reply:
You're right @ryantxu. Originally, I was also upgrading the InfluxDB Go client, but I ended up doing that in another PR to fix a serious bug of leaking TCP connections.

While I was working on the code, I figured I could also tighten up the code.